### PR TITLE
Update development strategy

### PR DESCRIPTION
**Pull request by markusdregi**
Updates to the development strategy after discussions.

The second commit moves the list of milestones into a separate section of the documentation to make them more accessible...

### CODE REVIEW

**Review by sondreso — APPROVED** [review]
LGTM, this should be rebased after https://github.com/equinor/ert/pull/2455 is merged to ensure there are no rst warnings :memo:

File: docs/rst/manual/developer_documentation/dev-strategy.rst [hunk 1]
```diff
@@ -88,59 +88,55 @@ This section contains a list of epics suggested to be carried out to continue
...
+serialization of record data.
 
-Run ERT2 forward models using prefect evaluator
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Transpile the ERT2 forward model into a single Unix Step that can be executed
-by the prefect evaluator. This should be introduced via a feature flag, that is
-later made default, before the legacy evaluator is removed.
+Act upon feedback on new data visualiser
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Actively seek feedback on webviz-ert and act upon the feedback such that
```
Comment (oyvindeide) [inline docs/rst/manual/developer_documentation/dev-strategy.rst]: Does not really belong here perhaps, but should we add a word about `everviz`?

Comment (markusdregi) [reply docs/rst/manual/developer_documentation/dev-strategy.rst]: I suggest we do it when start the milestone instead :) This is not the scout milestones, it is the ert milestones...

**Review by oyvindeide — APPROVED** [review]
LGTM

**Pull request merged**